src/components/LandingPage.jsx

```diff
@@ -153,8 +161,13 @@ const LandingPage = ({history}) => {
                 </Col>
             </Row>
 
-            <div style={{position: 'absolute', right: '110px', top: '85px', marginBottom: "100px"}}>
-                {getCard(details)}
+            <div style={{position: 'absolute', right: '110px', top: '85px', marginBottom: "100px"}} >
+                <DeveloperCreditsCard cardStatus={details=="developerCredits"}/>
+                <ResearchGroupCard cardStatus={details=="researchGroups"}/>
+                <CitationCard cardStatus={details=="citation"}/>
+                <RepositoriesCard cardStatus={details=="repositories"}/>
+                <SpecialCreditsCard cardStatus={details=="specialCredits"}/>
+                <ContactAndIssuesCard cardStatus={details=="contact"}/>
```

Comment:
Check out my comment above. We can revert back to `{getCard(details)}`, as this function will update the card displayed when `details` changes it's state.